packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Switch.module.scss';
+import { enableFocusStyles, disableFocusStyles } from './_SwitchUtils';
+
+const cx = classNames.bind(styles);
+
+// Translations Required
+const SWITCH_STATE = Object.freeze({
+  ON: 'On',
+  OFF: 'Off',
+});
+
+const propTypes = {
+  /**
+   * Id of the Switch button.
+   */
+  buttonId: PropTypes.string.isRequired,
```

Comment:
Could we rename this to `switchId`

Reply:
renamed it with this [commit](https://github.com/cerner/terra-core/pull/2846/commits/79847572906fa48acb184d1372057df82eab7134)